### PR TITLE
Rs 2022.10.08.1

### PR DESCRIPTION
**Pull request by rschlaikjer**
Patched up some choices later in the game as well that fell afoul of the new lint rule. I read as little as the surrounding text as possible, so they might not be ideal, but I'll get a second crack at them later.

I also changed instances of vampires' here to vampire's, since Roa seems to be the main topic most of the time.

2022-10-08-185417_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 1]
```diff
@@ -715,62 +787,75 @@ Ciel-senpai starts talking to me, though it sounds like she's talking to herself
...
 -- 「ちょっ、この状況でホラー映画の話とか。脅かしっこはナシですよ、先輩」
-"W-Wait, telling a horror story in this situation, we shouldn't scare each other, Senpai."
+"W-Wait, don't go telling me a horror story in a place like this.
+ There's no need to try and scare me, Senpai."
 }
 [sha:08c7c8ae86d4cea15192d272775b60a8ee6acdc2]{
 -- Page 232, Offset 18848.
 -- 「たとえ話ですから最後まで聞いてください。
-"I'm talking about what-ifs, so please listen to me until the end.
+"It's just a what-if, so please hear me out.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:805]: 最後まで聞いてください could be something more along the lines of "please let me finish"

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 2]
```diff
@@ -715,62 +787,75 @@ Ciel-senpai starts talking to me, though it sounds like she's talking to herself
...
 }
 [sha:319b95121507306322aff3d432ed89e101201a18]{
 -- Page 232, Offset 18849.
 -- 遠野くんはたまたま、気まぐれで映画館によってその映画を観たとします。“誰か”が主役の、そのホラー映画を。見終わってふと横の席を見ると、そこには悪役だった“誰か”が座っていました。
-Tohno-kun, let's say that on a whim, you go to a theater and watch that movie: that horror movie where 'someone' played the main part. When the movie ends, you glance at the neighboring seat, only to find that 'someone' sitting there.
+Tohno-kun, let's say that on a whim, you go to a theater and watch that movie.
+ The horror movie in which 'someone' played the main part.
+ When the movie ends, you glance at the neighboring seat,
+ only to find that 'someone' sitting there.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:813]: Needs a bit tightening up, imo.
> The same horror movie in which that 'someone' played the main part.
> When the movie ends, you glance at the neighboring seat,
> only to find them sitting there.

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 3]
```diff
@@ -715,62 +787,75 @@ Ciel-senpai starts talking to me, though it sounds like she's talking to herself
...
 }
 [sha:601460289443f2c141d0c13a20596345e6864178]{
 -- Page 232, Offset 18850.
 -- 映画のエンドロールから抜け出してきたばかりの、血まみれの“誰か”が」
-That certain 'someone', blood-soaked, who just came out from the ending credits."
+They've just slipped away, covered in blood, out from the end credits of the movie."
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:818]: > Having just slipped away from the end credits of the movie, this 'someone' is right next to you, covered in blood.

I think the dramatic structure of the sentence (with the 血まみれの“誰か”が at the end) needs a little preserving.

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 4]
```diff
@@ -715,62 +787,75 @@ Ciel-senpai starts talking to me, though it sounds like she's talking to herself
...
 -- Page 233, Offset 18851.
 -- 「彼は言います。
-"He says.
+"He says:
 }
 [sha:e144b2faee9548d5b297b8128660b5dbb4c1a57c]{
 -- Page 233, Offset 18852.
 -- “はじめまして誰かさん。映画の世界からやってきたものです。ああでも安心して、怖がらないで。私は台本通りに殺人鬼を演じただけ。それがまさか、終わった後に映画から追い出されるとは思ってもいなかった”」
-'Please to meet you, Mr. Stranger. I came from the world of that movie. Ah, but please don't be afraid, don't be nervous. I was just playing the role of the killer according to the script. Well, I certainly didn't expect to be thrown out of the movie as soon as it ended.'"
+'Pleased to meet you, Mr. Stranger.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:828]: はじめまして**誰かさん**  being the same 誰か as before certainly is meaningful, but I find it hard to pinpoint how.

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 5]
```diff
@@ -715,62 +787,75 @@ Ciel-senpai starts talking to me, though it sounds like she's talking to herself
...
 -- それとも―――映画の中に戻れと批難するか。
-Or perhaps――― Do you blame him, telling him to go back inside the movie?
+Or――would you indignantly demand that he go back inside the movie?
 }
 [sha:320c7cfee9c2c58e8e206c5d96ef7eb2710bd416]{
 -- Page 234, Offset 18857.
 -- この場合、どれが人間として正しいのでしょう？」
-What is, in this situation, the right choice to make as a human?"
+// Buddy you are asking the wrong guy
+In this situation, what is the right choice to make as a human?"
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:858]: > In this situation, what is the decent, human choice?

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 6]
```diff
@@ -785,52 +870,62 @@ I can't come up with an immediate answer.
...
+ It felt so incredibly real that I wanted to share that horror with you."
 }
 [sha:70722b4ab998f0c55529b0adbe5e30869a5b0f7b]{
 -- Page 238, Offset 18866.
 -- 「そ―――そうですか。とつぜん意地が悪くなったコトはどうかと思いますけど、その映画自体は面白そうですね。二重構造でよく出来ています。タイトルを教えてくれれば俺も借りてみますよ」
-"I-Is that so? I don't know what to think of you making fun of me, but the movie itself seems to be interesting. The meta-narrative is nicely done. Tell me the title and I'll rent it too."
+"I-Is that so?
+ I'm not sure how I feel about you picking on me out of the blue like that,
+ but the movie itself sounds interesting.
+ It's a good use of a meta-narrative.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:911]: > That's a well crafted meta-narrative.

**Review by Petrikow — COMMENTED** [review]
otu

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 7]
```diff
@@ -352,27 +396,31 @@ In other words, that means...
...
+ but Senpai is once again smiling like she usually does.
+ Yeah, this really does suit her best.
 }
 [sha:d3b273ce08d31fafdfd97eb2ad700fde018cbdde]{
 -- Page 207, Offset 18775.
 -- 「―――ただし。その前に反省してもらわなければいけません。貴方がどれほど度し難い失敗をしたのか、分かっていますか？」
-"―――However, I'll have you apologize first. Do you even realize how irredeemable the mistakes you made were?"
+"However.
+ Before we go, I need to hear an apology from you.
+ Do you even realize how grave the mistakes you made were?"
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_2_1.txt:408]: ofc not, but it'd become "Uh... what mistake?" or "Uh... when/how did I mess up?" or something like that.

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 8]
```diff
@@ -575,62 +641,68 @@ We enter the abandoned hospital.
...
 -- 「じゃあ遠慮なく。
-"Then I'll ask away.
+"Alright, here goes.
 }
 [sha:b28ec83d64a457d96dcac8616e9bcafc472d0ad2]{
 -- Page 223, Offset 18824.
 -- 吸血鬼は二種類いて、先輩たちがやっつけてるのは死徒の方ですよね。先輩たちが追っていた死徒……この街に居着いた死徒はどんなヤツだったんですか？」
-There are two types of vampires, and the ones you're pursuing are Dead Apostles. The Dead Apostles that you've chased, those that settled in this city, what are they like?"
+There are two types of vampires, and the one you're after is one of the Dead Apostles, right?
+ This Dead Apostle you've been chasing... the one that's made their nest in our city, what are they like?"
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_2_1.txt:678]: I feel like it makes sense for Shiki to use humanizing language for vampires - he is, after all, specifically conflicted about their 人間らしくness in either this file or one of the adjacent ones

**Pull request merged**